doc/adr/0011-separation-eid-implementation.md

```diff
@@ -0,0 +1,29 @@
+# 11. eID implementation in separate Microservice (Java based server due to Java SDK) and additional custom API for eService
+
+Date: 2022-07-11
+
+## Status
+
+Accepted
+
+## Context
+
+There are 2 types of online administration website (eService) available
+
+1. eService with eID integration (Magnus Flow)
+2. eService without eID integration (Paula flow)
+
+This document is a decision record for any eService who wishes to use eID with our solution (Paula Flow).
+
+## Decision
+
+1. We separate the identification logic as a standalone backend rather than embedding the logic in the eService backend.
+2. We build a Java-based backend because we are going to utilize Java SDK from the eID server provider.
+3. The microservice should work as an intermediate or proxy server without storing any personal data.
+4. The microservice should provide an interface for the eService to initiate identification securely.
```

Comment:
Sorting by number might lead to distractions. 
```suggestion
- We separate the identification logic as a standalone backend rather than embedding the logic in the eService backend.
- We build a Java-compatible backend because we are going to utilize the Java SDK from the eID server provider.
- The microservice should work as an intermediate or proxy server without storing any personal data.
- The microservice should provide an interface for the eService to initiate identification securely.
```